src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -1464,7 +1465,15 @@ private void DisposeErrorBubble()
 
         public void UpdateBubbleContent()
         {
-            if (DynamoViewModel == null) return;
+            if (NodeModel.BlockInfoBubbleUpdates)
```

Comment:
You can simplify this with:
 if (NodeModel.BlockInfoBubbleUpdates == true || DynamoViewModel == null) return;

Reply:
I find it easier to read this way.